cmd/klotho/main.go

```diff
@@ -83,11 +84,7 @@ func main() {
 
 	err := root.Execute()
 	if err != nil {
-		if cfg.internalDebug {
-			zap.S().Errorf("%+v", err)
-		} else if !root.SilenceErrors {
```

Comment:
Doesn't this regress the silence errors (which we set on line 270 so it doesn't double-print)?

Reply:
nope, we still set that to make sure that we dont double print anything

Reply:
yeah we actually will want this so that the non plugin errors surface, going to add it back